src/promptbase/drop/drop.py

```diff
@@ -31,52 +31,60 @@ def extract_valid_answers(validated_answers):
 
     return answers
 
-with open(DROP_DATASET_PATH, "r") as f:
-    data = json.load(f)
 
+computed_idxs = set()
 prompts = []
 answers = []
-for key in data:
```

Comment:
This isn't really a deletion, but stashing the code in a function so that it's not run when the module is imported (since downloading the data fails). For now, I think this is the quickest way to get things in a testable state

Reply:
Thanks, this was a nice way to clean up that pattern from when we ran the scripts independently.

Reply:
It's an extremely ugly, quick hack so that I could actually run the package.